packages/tables/src/styled/StyledOverflowButton.ts

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { ThemeProps, css, DefaultTheme } from 'styled-components';
+import stripUnit from 'polished/lib/helpers/stripUnit';
+import { retrieveComponentStyles, DEFAULT_THEME, getColor } from '@zendeskgarden/react-theming';
+import { getRowHeight } from './StyledRow';
+import { SIZE } from './StyledTable';
+
+const COMPONENT_ID = 'tables.overflow_button';
+
+export interface IStyledOverflowButtonProps {
+  isHovered?: boolean;
+  isActive?: boolean;
+  isFocused?: boolean;
+  size: SIZE;
+}
+
+const getOverflowButtonSize = () => {
+  return '2em';
+};
+
+const colorStyles = (props: IStyledOverflowButtonProps & ThemeProps<DefaultTheme>) => {
+  const baseColor =
+    props.isHovered || props.isActive || props.isFocused
+      ? getColor('neutralHue', 800, props.theme)
+      : getColor('neutralHue', 600, props.theme);
+
+  return css`
+    color: ${baseColor};
+
+    &:focus {
+      outline: none;
+    }
+
+    &:active,
+    &:hover {
+      opacity: 1;
+      color: ${getColor('neutralHue', 800, props.theme)};
+    }
+
+    &[data-garden-focus-visible] {
+      background-color: ${getColor('neutralHue', 600, props.theme, 0.35)};
+    }
+  `;
+};
+
+/**
+ * 1. Reset for <button> element
+ * 2. Reset for <a>nchor element
+ */
+export const StyledOverflowButton = styled.button.attrs<IStyledOverflowButtonProps>({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION,
+  type: 'button'
+})<IStyledOverflowButtonProps>`
+  display: block;
+  transition: opacity 0.25s ease-in-out, background-color 0.1s ease-in-out;
+  opacity: ${props => (props.isHovered || props.isFocused || props.isActive ? '1' : '0')};
+  z-index: ${props => (props.isActive ? '1' : '0')};
+  margin-top: calc(${props => stripUnit(getRowHeight(props)) / 2} - 1em);
```

Comment:
Looks like `margin-top` is overwritten by the `margin-top` located in [`StyledHeaderRow`](https://github.com/zendeskgarden/react-components/pull/560/files#diff-9bd59fa289b5b864cfd28e9eb386056eR35). Could `margin-top` be removed here?

Reply:
That overridden value is only for when the `OverflowButton` is used within a `HeaderRow`. This value is the default for other usages within the body.

Reply:
Upon a second review, I noticed that `margin-top` also generates an invalid CSS property. Could the `math()` polished helper be used to retain the unit for the first value of the `calc` function?

<img width="1025" alt="Screen Shot 2020-01-07 at 1 17 54 PM" src="https://user-images.githubusercontent.com/1811365/71936480-63d7df00-315e-11ea-9876-e51b4bc1d28a.png">

Reply:
I had a few of these `calc()` fixes stashed and forgot to add back in. I've added a `math()` call to address this.